models/articles_model.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../db/connection");
+
+async function fetchAllArticles() {
+  const articles = await db.query(
+    `SELECT articles.author,title,articles.article_id,articles.topic,articles.created_at,articles.votes,article_img_url, COUNT(comments)::INT AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC`
+  );
+  return articles.rows;
+}
+
+async function fetchArticle(article_id) {
+  const id = Number(article_id);
```

Comment:
I believe this is a redundant line as the next think you do is insert it into a string literal